core/station/api/spec.did

```diff
@@ -1177,6 +1183,12 @@ type ListRequestsResult = variant {
 type GetRequestInput = record {
   // The request id to retrieve.
   request_id : UUID;
+  // Fill in all the additional info about the request operation, request types such as Call External Canisters will
+  // include the request argument and reply blobs, this can be a large amount of data and could potentially exceed the
```

Comment:
> and reply blobs

does it mean that `execution_method_reply` in `CallExternalCanisterOperation` is now gonna be null if `with_full_info` is set to `opt false`?

Reply:
and is this the reason why null defaults to `opt true` to avoid a breaking change (it is still a breaking change due to potentially exceeding size limits)

Reply:
as mentioned aboved, i'll update the default to be `false`, but execution_method_reply is still going to be returned, this will keep things backwards compatible.